appstudio-utils/util-scripts/lib/fetch/data.sh

```diff
@@ -16,14 +16,49 @@ json-data-file() {
   # prepared but not actually used but that's okay.
   mkdir -p "$dir"
 
-  file="$dir/data.json"
+  local file="$dir/data.json"
 
   # Better not silently overwrite data
   [[ -f $file ]] && echo "Name clash for $file!" && exit 1
 
   echo "$file"
 }
 
+# Emulate in-place editing with jq
+jq-in-place-edit() {
+  local jq_filter="$1"
+  local file="$2"
+  local tmp_file="$(mktemp)"
+
+  jq "$jq_filter" "$file" > "$tmp_file"
```

Comment:
Not sure we're handling errors here (similar to #84), to be on the safe side I'd do:
```suggestion
  jq "$jq_filter" "$file" > "$tmp_file" || return 1
```

Reply:
I went for this:

```
 33   jq "$jq_filter" "$file" > "$tmp_file" && \                                                                                                                                                                                             
 34     mv "$tmp_file" "$file" 
```